shared/params/config.go

```diff
@@ -0,0 +1,8 @@
+package params
```

Comment:
that's another global params struct... 
If we have an eth1 params struct let's put it there.

Reply:
will be removed in https://github.com/ethereum/eth2-ssv/pull/80